src/webgl/p5.RendererGL.js

```diff
@@ -575,6 +575,7 @@ p5.RendererGL = class RendererGL extends Renderer {
     this.userFillShader = undefined;
     this.userStrokeShader = undefined;
     this.userPointShader = undefined;
+    this.userImageShader = undefined;
```

Comment:
above this, where every boolean variables are defined and set to false, you need to set there `this._drawingFilter = false;`

and at the function 
```js
filter(...args) {
// where the image call is present, above that set the 
this._drawingFilter = true;
this._pInst.image(fbo, ....)

Reply:
Alright @perminder-17 working on these changes.!